spec/lib/manageiq/api/common/headers_spec.rb

```diff
@@ -10,4 +10,14 @@
       expect { ManageIQ::API::Common::Headers.current = header_bad }.to raise_exception(ArgumentError)
     end
   end
+
+  context "Base64 encoding" do
+    it 'returns a Base64 representation of a hash' do
+      headers = { 'x-rh-auth-identity' => { 'identity' => { 'is_org_admin' => false } } }
```

Comment:
@syncrou This needs to be x-rh-identity based on https://docs.google.com/document/d/1PAzJqcUXlxg7t5cX1lsPsQtBPT_95bZbyB9Iiv_ekGM/edit#heading=h.9rvb6r5r6f4n